docs/UserGuide.md

```diff
@@ -79,45 +96,49 @@ Examples:
 * `help add`
 * `help edit`
 
-### Adding a person: `add`
-
-Adds a person to the address book.
+### Adding a person : `add`
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Adds a person to SocialBook.
 
+Format: `add n/NAME p/PHONE e/EMAIL a/ADDRESS dob/DATE OF BIRTH [pri/PRIORITY = LOW] [income/INCOME = 0] [famsize/FAMILY SIZE = 1] [r/REMARK] [t/TAG]...​`
+* Parameters can be inputted in any order.
+* Duplicated names are not allowed to be added.
+* Blank parameters are not allowed.
+* `NAME` are case-insensitive e.g. `n/JOHN DOE` is different from `n/john doe`.
+* `PHONE` should contain at least 3 digits.
+* For optional parameters like `PRIORITY, INCOME, FAMILY SIZE`, if not specified, their values will be defaulted to `LOW, 0, 1` respectively.
 <box type="tip" seamless>
 
 **Tip:** A person can have any number of tags (including 0)
 </box>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 dob/9 Mar 1999 famsize/3 income/5000`
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 dob/25 Dec 2002 pri/MEDIUM t/criminal`
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in the address book, automatically sorted by their priority from HIGH to LOW.
+Shows a list of all persons in SocialBook. This list is by default sorted based on priority from HIGH to MEDIUM to LOW.
 
 Format: `list`
 
 ### Editing a person : `edit`
 
-Edits an existing person in the address book.
+Edits an existing person's details in the SocialBook.
```

Comment:
Should we standardise references to SocialBook as "in SocialBook", so omit the "the" before SocialBook here?